partner_event/models/res_partner.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2014 Tecnativa S.L. - Pedro M. Baeza
+# Copyright 2015 Tecnativa S.L. - Javier Iniesta
+# Copyright 2016 Tecnativa S.L. - Antonio Espinosa
+# Copyright 2016 Tecnativa S.L. - Vicent Cubells
+# Copyright 2020 Tecnativa S.L. - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    event_registration_ids = fields.One2many(
+        string="Event registrations",
+        comodel_name="event.registration",
+        inverse_name="attendee_partner_id",
+    )
+    registration_count = fields.Integer(
+        string="Attendances", compute="_compute_registration_count", store=False
+    )
+
+    @api.depends("event_registration_ids")
```

Comment:
Instead of calling `invalidate_recordset()`, you must add here in this depends the subfields (like `event_registration_ids.state`) for triggering the recompute automatically, as this is also going to provoke errors on its usage.

Reply:
Done!

Reply:
The change is not correct. You have to add both,  `event_registration_ids` and `event_registration_ids.state`, and any other field that can impact, and remove the `invalidate_recordset` calls in the tests.

Reply:
Ok, I have applied the changes and everything works correctly.